### PR TITLE
adding global hotkey to show devtools to help debug. updating some packages

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: desktop/app/dockIcon.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import app from 'app'
+import {app} from 'electron'
```
Comment (chrisnojima) [inline desktop/app/dockIcon.js]: Dunno how this worked before...

File: desktop/package.json [hunk 2]
```diff
@@ -11,7 +11,6 @@
     "build-prod": "NODE_ENV=production webpack --config webpack.config.production.js --progress --profile --colors",
     "package": "NODE_ENV=production node package.js",
     "hot-server": "HOT=true NODE_ENV=production DEBUG='express:*' node server.js",
-    "postinstall": "./node_modules/.bin/electron-rebuild",
```
Comment (chrisnojima) [inline desktop/package.json]: This command is problematic a bunch...

File: desktop/app/dockIcon.js (hunk 1 above)
Comment (MarcoPolo) [inline desktop/app/dockIcon.js]: This is how it used to be

File: desktop/renderer/index.js [hunk 3]
```diff
@@ -75,8 +75,14 @@ class Keybase extends Component {
         return diffState
       }
 
-      sender.send('stateChange', getStore())
+      if (!sender.isDestroyed()) {
+        sender.send('stateChange', getStore())
+      }
       store.subscribe(() => {
+        if (sender.isDestroyed()) {
```
Comment (MarcoPolo) [inline desktop/renderer/index.js]: :sheep: Not sure if it's a good idea to also unsubscribe from the store? store.subscribe returns a function you can call to unsubscribe the listener.

**Comment by MarcoPolo** [comment]
:+1: % optional :sheep:

File: desktop/app/dockIcon.js (hunk 1 above)
Comment (chrisnojima) [inline desktop/app/dockIcon.js]: not since 0.35... right?

File: desktop/package.json [hunk 4]
```diff
@@ -48,11 +40,15 @@
     "redux-thunk": "^1.0.3"
   },
   "devDependencies": {
+    "babel": "^5.8.34",
```
Comment (chrisnojima) [inline desktop/package.json]: these are actually dev depenencies

File: desktop/app/dockIcon.js (hunk 1 above)
Comment (chrisnojima) [inline desktop/app/dockIcon.js]: I think they depecated it finally so it no longer works in newer electron

**Pull request merged**